core/row/row.selector.d.ts

```diff
@@ -0,0 +1,16 @@
+import {Model} from 'ng2-qgrid/core/infrastructure/model';
+import {ColumnModel} from 'ng2-qgrid/core/column-type/column.model';
+
+export declare class RowSelector {
+	constructor(model: Model);
+
+	model: Model;
+	source: string[];
+
+	map(items: any[]): void;
+	mapFromRows(rows: any[]): (rows: any[], columns: any[]) => any;
+	mapFromColumns(columns: any[]): (rows: any[], columns: any[]) => any;
```

Comment:
Looks that ```columns``` is array of ```ColumnModel``` in these methods

Reply:
right)